app/models/cdo/create/address.js

```diff
@@ -46,34 +53,32 @@ function ViewModel (address, errors) {
       autocomplete: forms.preventAutocomplete,
       attributes: { maxlength: '8' }
     },
-    country: {
-      id: 'country',
-      name: 'country',
-      label: {
-        text: 'Country'
-      },
-      value: address.country,
-      autocomplete: forms.preventAutocomplete,
-      attributes: { maxlength: '30' }
-    },
+    countries: mapToCountrySelector(countries, validationError, address.country),
     errors: []
   }
 
-  if (errors) {
-    for (const error of errors.details) {
+  if (validationError) {
```

Comment:
If you're changing the name of 'errors' to '#validationError', I don;t have a problem - but it should be plural (i.e. 'validationErrors') as there may be more than one in the array

Reply:
As discussed, this is a Joi.ValidationError object containing an array of details.